types/error.go

```diff
@@ -1,7 +1,15 @@
 package types
 
-import "errors"
+import (
```

Comment:
Do we wanna have all errors in a single file again?

Reply:
No no, this is the sdk.error type that we will use throughout our app!